setup.py

```diff
@@ -124,32 +124,45 @@ def find_version(*file_paths):
                     "wheel>=0.36.1",
                     "openvino-telemetry"]
 
-
-EXTRAS_REQUIRE = {
-    "tests": ["pytest"],
-    "docs": [],
-    "tf": [
+TF_EXTRAS = [
         "tensorflow~=2.8.4",
-    ],
-    "torch": [
+    ]
+
+TORCH_EXTRAS = [
         "torch>=1.8.2,<1.14",
-    ],
-    "onnx": [
+    ]
+
+ONNX_EXTRAS = [
         "onnx==1.12.0",
         "onnxruntime-openvino==1.13.1"
-    ],
-    "openvino": [
+    ]
+
+OPENVINO_EXTRAS = [
         "openvino-dev~=2022.2.0"
```

Comment:
Unrelated to this PR, but could this changed to 2022.3.0?

Reply:
I rebased this PR on top of develop, and the requirement has been relaxed there already.